ext/mbstring/tests/mb_str_split_ru.phpt

```diff
@@ -0,0 +1,57 @@
+--TEST--
+mb_str_split() tests for the japanese language
```

Comment:
This is not Japanese, but Russian, I believe?

Reply:
it is.